### DISCUSSION TITLE
How to change theme for only a container and it's all children

### DISCUSSION THREAD

**Discussion by thekaganugur** [chakra-ui/chakra-ui#2768] [Q&A]
Hi,
You can easily understand my problem by looking [this codesandbox]( https://codesandbox.io/s/friendly-maxwell-lk9by?file=/src/App.tsx).

Simply, I need a Component that has a white background when the app is still in dark mode. I tried `<LightMode>` but it does not work.

**Comment by franky47** [comment]
Could you change this component's background directly ?

```tsx
<YourComponent bg="white">
  <LightMode>
    // The rest of your tree that will render in light mode
  </LightMode>
</YourComponent>
```

Changing the background color in the theme will not help here, as the general background color is that of `<body>` and cannot be changed for a single component (while remaining the same for the rest of the page).

**Reply by thekaganugur** [reply]
Thank you for your response, did you have a look at the sandbox? What you said is from what I can see exactly what I tried.

```
      {/* Everything should work but does not */}
      <LightMode>
        <Box bg="white" mt="10" h="300px">
          <Heading>Not working heading</Heading>
          <Heading color="black">Fixed by color prop heading</Heading>
          <Input placeholder="Basic usage" />
        </Box>
      </LightMode>
```

But it does not work
